docs/DeveloperGuide.md

```diff
@@ -1509,11 +1624,54 @@ Character limit:
 
 1. Dealing with missing/corrupted data files
 
+<<<<<<< HEAD
+    1. **Prerequisites**: At least one modification has been made to the persons list. FAST is not currently open.
+    
+    2. **Test Case**: 
+        1. Go to the directory that contains FAST.jar.
+        2. navigate to `data` and open `fast.json`.
+        3. In line 3, change "name" to "test".
+        4. Launch FAST. <br>
+       
+        **Expected**: FAST should be completely blank, with no data displayed.
+        
+    3. **Test Case**:
+        1. Go to the directory that contains FAST.jar.
+        2. navigate to `data` and delete `fast.json`.
+        3. Launch FAST. <br>
+    
+        **Expected**: FAST should contain a default set of persons.
+
+    
+=======
```

Comment:
Similarly, you might have forgotten to remove this line after resolving the merge conflict